### ISSUE TITLE
Tutorial 1 not working with GridLAB-D

### ISSUE THREAD

**Issue by trevorhardy** [GMLC-TDC/HELICS-Tutorial#13]
GridLAB-D now returns a complex data type instead of a real, imaginary tuple which federate1.py is expecting. The example should be updated to handle the new output from GridLAB-D.

GridLAB-D version: feature/730
HELICS version: v3.3.0

l.89 in federate1.py
**is:** `rValue, iValue = h.helicsInputGetComplex(subid)`
**should be:**  cValue = h.helicsInputGetComplex(subid)

**Comment by afisher1** [comment]
I don't think this is a cause of GridLAB-D. The python HELICS api functions concerning complex values were updated to accept and return python complex types around v3.0 of HELICS. It still doesn't change the recommended solution though.
